aws/hhvm1/state-machine/generate.hack

```diff
@@ -327,19 +340,19 @@ function linear_state_machine(
  * Adds in the Succes/Failure states that we need in every nested state machine.
  */
 function nested_state_machine(
-  string $prefix,
-  ?P $add_param,
+  string $end_state_prefix,
+  ?P $add_param_to_results,
```

Comment:
I forgot what these were for and I only wrote it, like, last week, so I figured I should give them better names.